interoptest/src/javaservice/src/main/java/io/opencensus/interop/ServiceHopper.java

```diff
@@ -42,6 +45,9 @@
 final class ServiceHopper {
   private static final Logger logger = Logger.getLogger(ServiceHopper.class.getName());
   private static final Tagger tagger = Tags.getTagger();
+  private static final Tracer tracer = Tracing.getTracer();
+  private static final String ID_ATTRIBUTE_KEY = "req.id";
```

Comment:
This should be "reqId". See https://github.com/census-ecosystem/opencensus-experiments/pull/165/files#diff-52c50f72a64a669802062f7b2a245210R84, https://github.com/census-ecosystem/opencensus-experiments/pull/163/files#diff-144f321644819dda858f64caf79690aeR33 and https://github.com/census-ecosystem/opencensus-experiments/pull/156/files#diff-58a54f6d89e9a5c8d24f0e01926c7644R57.

Reply:
I based it on what was in the spec but, after consulting with Rahul, I've updated it (he will update the spec in another pr).